benchmark/opperf/utils/profiler_utils.py

```diff
@@ -248,12 +248,11 @@ def python_profile(func):
     @functools.wraps(func)
     def python_profile_it(*args, **kwargs):
         runs = args[1]
-        modified_args = (args[0], 1, args[2])
         times = []
 
         for _ in range(runs):
             start_time = time.perf_counter()    # 1
-            res = func(*modified_args, **kwargs)
```

Comment:
@connorgoggins @apeforest 
If we pass the *args as is, it will still have
args[0] as op
args[1] as runs
For eg if user passed runs as 10

So the native profiler would run 10 times and so will the for loop run 10 times (for timing Python profiler)

Coz the func here is nd_forward_backward_profile or nd_forward_profile (both take runs as a parameter)